build.gradle

```diff
@@ -40,8 +40,8 @@ subprojects { subProject ->
         apply plugin: "io.spinnaker.plugin.service-extension"
     }
 
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
I think the targetCompatibility should be JDK 11. Based on https://harness.atlassian.net/browse/ASSMT-560 but not sure if that will cause issues

Reply:
@jasonmcintosh based on the ticket Edgar posted should we do it target 11?

Reply:
Target should be JAVA 11 since we still publish for slightly older bases.  I'd LIKE to get all of this merged into OSS at SOME Point instead of using this plugin.  17 can use 11 targeted libraries for runtime just fine.

Reply:
will use target 11 for now :)

Reply:
Cause: warning: source release 17 requires target release 17

Reply:
Ugh, 11 both for now!